backend/routes/indexer.go

```diff
@@ -384,3 +390,16 @@ func processNewDayEvent(event IndexerEvent, w http.ResponseWriter) {
 		}
 	}
 }
+
+func sendNFTWebSocketMessage(tokenID string) {
+	message := map[string]interface{}{
+		"token_id":    tokenID,
+		"messageType": "nftMinted",
+	}
+	sendWebSocketMessage(message)
+}
+
+func sendWebSocketMessage(message map[string]interface{}) {
+	// Generic function to send a message over a WebSocket
+	// Implementation logic to send the message...
```

Comment:
For the implementation use the lines 148-151 here https://github.com/keep-starknet-strange/art-peace/blob/b330eece8d98a10e4bea128047c552f3e5d0910d/backend/routes/indexer.go#L148